crates/flux-middle/src/fhir/lift.rs

```diff
@@ -221,7 +221,7 @@ impl<'a, 'tcx> LiftCtxt<'a, 'tcx> {
     ) -> Result<fhir::WhereBoundPredicate, ErrorGuaranteed> {
         if let hir::WherePredicate::BoundPredicate(bound) = pred {
             if !bound.bound_generic_params.is_empty() {
-                return self.emit_unsupported("higher-rank trait bounds are not supported 3");
+                return self.emit_unsupported("higher-rank trait bounds are not supported");
```

Comment:
Btw, when you use `cargo xtask run` you'll see that we report the location an error was emitted, e.g.,

```
error[FLUX]: refinement type error
 --> attic/playground.rs:4:5
  |
4 |     0
  |     ^ a postcondition cannot be proved
-Ztrack-diagnostics: created at crates/flux-refineck/src/lib.rs:114:15   <------- this
```
You can also pass `-Ztrack-diagnostics=y` to enable it if you are not using `cargo xtask run`

Reply:
aha good to know -- does it apply to `panic` etc too? should add to the docs

Reply:
Both `-Ztrack-diagnostic` and `panic` behavior depends on whether functions are annotated with [`#[track_caller]`](https://doc.rust-lang.org/reference/attributes/codegen.html#the-track_caller-attribute). I typically add it to functions when I encounter the errors and I think it will make the error more useful. For `panic` though you can always get the full story enabling the backtrace.